clrdefinitions.cmake

```diff
@@ -226,3 +226,7 @@ add_definitions(-DFEATURE_WINMD_RESILIENT)
 add_definitions(-D_SECURE_SCL=0)
 add_definitions(-DUNICODE)
 add_definitions(-D_UNICODE)
+
+if(FEATURE_CROSSBITNESS)
+  add_definitions(-DFEATURE_CROSSBITNESS)
```

Comment:
Do we really need this? I would prefer not to add this define. We already know the host and target architectures, and this is a function of that. Individual components can define this if absolutely required.

Reply:
Well, I thought that since it will be consumed by other components (not only jit, but also vm and zap) I define the feature and the macro here and avoid repeating the same `#if defined(_TARGET_ARM_) && defined(_HOST_AMD64)_ ` in headers.

Reply:
While I like consistency, I really dislike adding every possible computable definition to the global build rules. There are far, far too many of these already. I would ideally like the JIT to build self-contained without requiring the definition of so many external things.